src/pages/Settings/EmailSettings.vue

```diff
@@ -0,0 +1,518 @@
+								<option value="monthly">
+									Send a monthly digest
+								</option>
+							</kv-dropdown-rounded>
+						</fieldset>
+						<!-- TODO Implement Service Worker for push notifications -->
+						<!-- When repaymentUpdates == 'none' this should be hidden -->
+						<!-- <div class="input-wrapper nested">
+								<input id="push-repayment-updates-input" type="checkbox">
+								<label for="push-repayment-updates-input">
+									Receive notifications on this device
+								</label>
+							</div> -->
+						<fieldset>
+							<label for="autolend-updates-input">Autolending notifications</label>
+							<kv-dropdown-rounded
+								id="autolend-updates-input"
+								v-model="form.autolendUpdates"
+							>
+								<option value="none">
+									Do not send
+								</option>
+								<option value="nightly">
+									Send a nightly digest
+								</option>
+								<option value="monthly">
+									Send a monthly digest
+								</option>
+							</kv-dropdown-rounded>
+						</fieldset>
+						<fieldset>
+							<a href="/lend/saved-search">Saved searches notifications</a>
+						</fieldset>
+					</form>
+				</template>
+			</kv-settings-card>
+		</div>
+
+		<div class="row">
+			<!-- Loan Settings -->
+			<kv-settings-card
+				class="columns small-12 large-8"
+				title="Lending activity updates"
+				:disabled="form.globalUnsubscribed"
+			>
+				<template #content>
+					<form @submit.prevent>
+						<fieldset>
+							<kv-button class="text-link" @click.native="lendingToggleAll()">
+								{{ lendingAllSelected ? "Deselect" : "Select" }} All
+							</kv-button>
+						</fieldset>
+						<fieldset>
+							<kv-checkbox
+								id="loanUpdates"
+								name="loanUpdates"
+								v-model="form.loanUpdates"
+								class="email-settings__checkbox"
+							>
+								Loan and borrower updates
+							</kv-checkbox>
+						</fieldset>
+						<fieldset>
+							<kv-checkbox
+								id="commentsMessages"
+								name="commentsMessages"
+								v-model="form.commentsMessages"
+								class="email-settings__checkbox"
+							>
+								Loan comments, lender to lender messages
+							</kv-checkbox>
+						</fieldset>
+						<fieldset>
+							<label for="team-digests-input">Team digests</label>
+							<kv-dropdown-rounded
+								id="team-digests-input"
+								v-model="form.teamDigests"
+							>
+								<option value="no">
+									Do not send
+								</option>
+								<option value="yes">
+									Send as they arrive
+								</option>
+								<option value="nightly">
+									Send a nightly digest
+								</option>
+								<option value="weekly">
+									Send a weekly digest
+								</option>
+							</kv-dropdown-rounded>
+						</fieldset>
+					</form>
+				</template>
+			</kv-settings-card>
+		</div>
+
+		<!-- !TODO Add Trustee email settings section  -->
+		<!-- !TODO Add Borrower email settings section  -->
+		<div class="row">
+			<div class="columns small-12 large-8">
+				<kv-button
```

Comment:
I think semantically it would be make sense to have all of these settings cards wrapped in one single form. Then this button would be a type="submit" and wouldn't need @click.native. This would also allow you to hit "enter" while you're editing the form to submit it.

Roughly
`<form @submit.prevent="saveSettings" :disabled="!isChanged || isProcessing">` 
and 	
`<kv-button class="smallest" type="submit" :disabled="!isChanged || isProcessing">`

Reply:
Wrapped it in a single form, I agree... I tried making the button a submit button, but ran into an unfortunate issue - Changing the first drop down, focus seems to move to the "Select All" button, hit enter, the form then both activates "Select All" and submits. 
So I left the default form submit null and kept the manually firing of the submit function on the button